homework/Evgeny_Korovin/Hw6_Task5/src/main/java/by/evgKor/service/PowerCalculatorService.java

```diff
@@ -0,0 +1,16 @@
+package by.evgKor.service;
+
+public class PowerCalculatorService implements Runnable {
+    public static int exponentiation(int number, int power){
+        for(int i=number;i<power;i++){
+            System.out.println("Result: " + (int)Math. pow(number, power));
```

Comment:
try to change "power", for example... on 8, and  look what will happen. 
And program should work only with positive numbers, missing condition.

Reply:
Yes, corrected